CharacterCreation/02_Classes.txt

```diff
@@ -416,6 +417,24 @@ Skill Proficiencies:
 * Spotting
 * Survival
 
+== Hunter's Dog ==
+
+    When your character ascends to level 3, you receive an attack dog. Maybe 
+you found the dog along your last mission, or maybe a dog was issued to you when
+you returned to home base (at the GM's choice). Every time you level up after
+you have your dog, your dog levels up as well and learns a new "Trick" as 
+defined in the "Class-Specific Documentation/Dog_Tricks.txt" document. For 
+simplicity, Dogs start at level 3 so that they are always the same level as 
+their owners. 
+
+    Your new dog is well-trained and responds to a variety of commands. Your dog 
+listens to you and does his best to make his or her owner happy, but is not 
+controlled by the player. Instead, the owner character commands the dog, and the
+GM controls how the dog responds. 
```

Comment:
ahah, I do very much too. Further more I worry very much about a new player projecting a beloved pet onto the dog and absolutely losing it if the dog dies in the line of duty. I've seen something similar go horribly wrong in another RPG back in high school. 

I'm so concerned with these two issues that I'm considering halting work on Hunter and leaving it how it is. Or maybe doing only minor touch-ups and scrapping the dog idea altogether.

Reply:
The work flow problem at least we might ameliorate by giving the player nominal control over the pet but with explicit DM veto power.

As for the other bit, would it be enough to include a projection warning in our DMG-equivalent 'handling players' section?

Reply:
Oh, another thought. Maybe give the dog bleeding out rules like PCs so it can be saved if it goes down.

Reply:
My intention is to give specific language saying that the dog is well-trained and friendly to both the handler and their teammates, to prevent knuckle-heading "My dog is antagonizing you, but it's not me or my character" nonsense. I think we'll need warnings in both the DM guide and Player handbook on all of these issues, including the potential danger for the dreaded DMPC-diva.

Reply:
I agree that these design problems are solvable. but I would liken it to flying a little close to the sun. It's interesting and unique, but volatile.

Reply:
@rosebrallier How did Critical Role handle the friendly Trinket?

Reply:
What's wrong with 5e ranger w/ pet? (I honestly don't know). Avoiding whatever they did wrong sounds like a good design choice.

Reply:
In 5e, the pet scales terribly. At higher levels (like 10+), it's attack rolls, hp and dmg become inconsequential and unimpactful to the point where it's attacks (which require the player's action) simply can't compete even with the ranger's underpowered spells. The pet can generate sneak attack (flanking) bonuses and not much else. Also, the pet has no out-of-combat function.

Reply:
At the risk of hubris, I think I can out-design that handily. I will not be using 5E (nor 4E) as a template.

Reply:
Excellent